src/installers/darwin/dmg.js

```diff
@@ -1,5 +1,31 @@
-import opn from 'opn';
+import { spawn } from 'child_process';
```

Comment:
After some playing it looks like none of the methods in `spawn-rx` run this command correctly and result in the same action as `child_process`.  I.e. only when using `child_process` does the finder window appear.

/cc @paulcbetts